cmd/oceantv/broadcast_states.go

```diff
@@ -307,26 +307,49 @@ func newDirectLive(ctx *broadcastContext) *directLive {
 	return &directLive{ctx}
 }
 func (s *directLive) enter() {}
-func (s *directLive) exit() {
-	err := s.man.StopBroadcast(context.Background(), s.cfg, s.store, s.svc)
-	if err != nil {
-		s.log("could not stop broadcast")
-	}
-}
+func (s *directLive) exit()  {}
 
 type directLiveUnhealthy struct {
 	*broadcastContext `json: "-"`
+	LastResetAttempt  time.Time
+	Attempts          int
 }
 
 func newDirectLiveUnhealthy(ctx *broadcastContext) *directLiveUnhealthy {
-	return &directLiveUnhealthy{ctx}
+	return &directLiveUnhealthy{broadcastContext: ctx}
 }
 func (s *directLiveUnhealthy) enter() {}
-func (s *directLiveUnhealthy) exit() {
-	err := s.man.StopBroadcast(context.Background(), s.cfg, s.store, s.svc)
-	if err != nil {
-		s.log("could not stop broadcast", s.cfg.Name, s.cfg.ID)
+func (s *directLiveUnhealthy) exit()  {}
+func (s *directLiveUnhealthy) fix() {
+	notify := func(msg string) {
+		notifier.SendOps(context.Background(), s.cfg.SKey, "health", msg)
 	}
+
+	const resetInterval = 5 * time.Minute
```

Comment:
Again, I'd prefer to these consts given more descriptive names (e.g., directLiveUnhealthyResetInterval.) and promoted to one place where all of them can be configured